chart/templates/loadbalancer.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: loadbalancer
```

Comment:
With a LoadBalancer type we will expose an external port. Is this want we want to do by default? I think by default we have an internal ip and ask people to port forward in the instructions.

Alternatively we could have a conditional in the `values.yaml` of the Helm Chart to enable a loadbalancer. That think that would be the most flexible thing. Thoughts? @samiyaakhtar @dennisseah

Reply:
That's a good idea @andrebriggs 

I can make this change. I'll also add some documentation on how to install via this helm chart